plenum/common/batched.py

```
@@ -127,6 +129,14 @@ def flushOutBoxes(self) -> None:
                              logMethod=logger.debug)
             del self.outBoxes[rid]
 
+    def _make_batch(self, msgs):
+        batch = Batch(msgs, None)
+        serialized_batch, _ = self.signAndSerialize(batch)
```

Do we ignore error (the second returned value)? Should we log it?